.orchestra/config/lib/create_component.lib.yml

```diff
@@ -37,17 +39,23 @@ builds:
     #@ if/end build_dependencies:
     build_dependencies: #@ build_dependencies
 
-    configure: #@ configure
-
+    configure: |
+      (@- if pre_configure: @)
+      (@-= pre_configure @)
+      (@ end -@)
+      (@-= configure -@)
+      (@- if post_configure: @)
+      (@= post_configure -@)
+      (@- end -@)
```

Comment:
I'm confused by `pre_configure` and `post_configure`. They don't seem to be used.

Reply:
I'll drop the commit, I added them because at some point I needed them for some component which is not part of this PR